### PR TITLE
ScreenSettings

### PR DESCRIPTION
**Pull request by Transliteration**

### CODE REVIEW

**Comment by Transliteration** [comment]
Вот тут должно стать понятно, зачем нужна лямбда!

File: Source/Screens/ScreenGame.cpp [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #include "ScreenGame.hpp"
 
-ScreenGame::ScreenGame()
+ScreenGame::ScreenGame(Settings &settings)
```
Comment (makometr) [inline Source/Screens/ScreenGame.cpp]: надеюсь, ссылки на Settings в конструкторах станут константными👍

Comment (Transliteration) [reply Source/Screens/ScreenGame.cpp]: Settings const &? Если так, мы не сможем менять настройки на экранах, объявленных таким образом. А Settings & const смысла не имеет. Я тебя не понял(

**Comment by Transliteration** [comment]
Ребят, предлагаю это дело слить в мастер, го?

File: CMakeLists.txt [hunk 2]
```diff
@@ -10,6 +10,7 @@ project(Tetris)
 file(GLOB srcMain Source/*.cpp)
 file(GLOB srcScreens Source/Screens/*.cpp)
 file(GLOB srcButtons Source/Buttons/*.cpp)
+file(GLOB srcSettings Source/Settings/*.cpp)
```
Comment (Denmey) [inline CMakeLists.txt:13]: Why separate globs? You can add all the source files like this:
`
file(GLOB src
   Source/*.cpp)
   Source/Screens/*.cpp)
   Source/Buttons/*.cpp)
   Source/Settings/*.cpp
)
`
Or even simplier:
`
file(GLOB_RECURSE src
   Source/*.cpp)
)
`
BUT! It's easy, but it's not recommended way to find source files like it's said in docs([cmake.org](https://cmake.org/cmake/help/v2.8.11/cmake.html#command:file)):

> GLOB will generate a list of all files that match the globbing expressions and store it into the variable. Globbing expressions are similar to regular expressions, but much simpler. If RELATIVE flag is specified for an expression, the results will be returned as a relative path to the given path. (We do not recommend using GLOB to collect a list of source files from your source tree. If no CMakeLists.txt file changes when a source is added or removed then the generated build system cannot know when to ask CMake to regenerate.)

You can use an easy way in this PR, but we have to find more appropriate way to do it in future.

**Pull request merged**